lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class MigrateDataToWorkField
+    QUERY = "data->>'employment_title' <> '' AND data->>'work' IS NULL".freeze
+
+    def run
+      Profile.where(QUERY).select(:id).find_each do |profile|
```

Comment:
I was tempted to use ` Profile.where(QUERY).ids` here at first (`select id from profiles where data->>'employment_title' <> ''` took about 6-7 seconds in Blazer) but then I thought I'd rather not instantiate an array of ~213k elements.